docs/develop/excel-async-model.md

```diff
@@ -133,10 +133,20 @@ async function main(context: Excel.RequestContext) {
 }
 ```
 
-## Office Scripts Async API reference documentation
+## Converting legacy async scripts to the current model
 
-[!INCLUDE [Async reference documentation](../includes/async-reference-documentation-link.md)]
+The current API model doesn't use `load`, `sync`, or a `RequestContext`. This makes the scripts much easier to write and maintain. Your best resource for converting old scripts is [Stack Overflow](https://stackoverflow.com/questions/tagged/office-scripts). There, you can ask the community for help with specific scenarios. The following guidance should help outline the general steps you'll need to take.
+
+1. Create a new script and copy the old code into it. Be sure not to include the old `main` method signature, using the current `function main(workbook: ExcelScript.Workbook)` instead.
+
+2. Remove all the `load` and `sync` calls. They are no longer necessary.
+
+3. All properties have been removed. You now access those objects through `get` and `set` methods. For example, instead of setting a cell's fill color through property access like this: `mySheet.getRange("A2:C2").format.fill.color = "blue";`, you'll now use methods like this: `mySheet.getRange("A2:C2").getFormat().getFill().setColor("blue");`
 
-## See also
+4. Collection classes have been replaced by arrays. The `add` and `get` methods of those collection classes were moved to the object that owned the collection, so your references must be updated accordingly. For example, to get a chart named "MyChart" from the first worksheet in the workbook, use the following code: `workbook.getWorksheets()[0].getChart("MyChart");`. Note the `[0]` to access the first value of the `Worksheet[]` returned by `getWorksheets()`.
 
-- [Improve the performance of your Office Scripts](web-client-performance.md)
+5. Some methods have been renamed for clarity and added for convenience. Please consult the [Office Scripts API reference](/javascript/api/office-scripts/overview?view=office-scripts) for more details.
```

Comment:
Is there a plan to do anything to add mapping for people in the reference? Might be helpful

Reply:
No, but this is fewer than a dozen methods that we have no evidence people are actually using in scripts. I'm not sure there's a record of the changes, so I can't justify the investigation cost.